src/Configuration/EditorConfig/EditorConfig.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\EasyCodingStandard\Configuration\EditorConfig;
+
+/**
+ * @see https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties
+ */
+class EditorConfig
+{
+    public function __construct(
+        public readonly ?IndentStyle $indentStyle,
```

Comment:
this seems cause downgrade build error, see

https://github.com/easy-coding-standard/easy-coding-standard/actions/runs/13073439464/job/36479848211#step:12:51

I will look into it.

Reply:
it seems due to enum usage

```php
enum IndentStyle: string
{
    case Space = 'space';
    case Tab = 'tab';
}
```

I think we can just use class and constant.

Reply:
I created new PR to fix it

- https://github.com/easy-coding-standard/easy-coding-standard/pull/260

by use class and constant instead.